salt/modules/boto_route53.py

```diff
@@ -83,7 +81,9 @@ def __virtual__():
     Only load if boto libraries exist.
     '''
     if not HAS_BOTO:
-        return (False, 'The boto_route53 module could not be loaded: boto libraries not found')
+        msg = ('A boto library with version at least {} was not '
```

Comment:
Could you please index this? `{0}` instead of `{}`. It will make the lint check happy. :]